### ISSUE TITLE
Compilation success depends on members order

### ISSUE THREAD

**Issue by valmat** [arcosuc3m/clang-contracts#19]
This example does not compile:
```c++
struct Q {
  void foo() [[expects: x==3]] {}
  int x = 42;
};
```
I have got following error:
```
error: use of undeclared identifier 'x'
  void foo() [[expects: x==3]] {}
                        ^
```
But if to change members order everything is fine:
```c++
struct Q {
  int x = 42;
  void foo() [[expects: x==3]] {}
};
```
compiles with no problems

**Comment by jalopezg-git** [comment]
This issue is a duplicate of #3, which depends on CXX11 attribute late parsing.  Closing this issue and leaving open #3 until ``[[expects]]`` and ``[[ensures]]`` are late parsed.
